code/datums/emergency_calls/xenoalpha.dm

```diff
@@ -0,0 +1,52 @@
+/datum/emergency_call/xenomorphs_alpha
+	name = "Xenomorphs (Alpha Hive)"
+	probability = 0
+	auto_shuttle_launch = TRUE
+	spawn_type = null
+
+
+/datum/emergency_call/xenomorphs_alpha/print_backstory(mob/living/carbon/xenomorph/X)
+	to_chat(X, "<B>You are Xenomorph from the Alpha hive.</b>")
+	to_chat(X, "<B>You've been cruising in space for years until a new Queen reached out to you and took over the control of your shuttle.</b>")
+	to_chat(X, "<B>Help the new Queen take over this sector. For the Alpha Hive!</b>")
```

Comment:
"Help" is wrong here

Reply:
Alright. Gonna change into something else then.